packages/angular-standalone-test-app/angular.json

```diff
@@ -45,7 +42,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "500kb",
-                  "maximumError": "1.1mb"
+                  "maximumError": "4.27mb"
```

Comment:
After using lazyComponent initial bundle size is smaller then 1mb, this field can be reduced to 1mb